google/resource_sql_database_instance.go

```diff
@@ -344,7 +344,36 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 					},
 				},
 			},
-
+			"server_ca_cert": &schema.Schema{
+				Type:     schema.TypeList,
+				Computed: true,
+				Optional: true,
```

Comment:
(hello friends I know this is already merged but-)
since this is only set on read (and not something passed in to create or update), it should be only computed and not also optional.

Reply:
thanks! follow up inbound